gbpservice/nfp/bin/startup/redhat/nfp_proxy.service

```diff
@@ -0,0 +1,11 @@
+[Unit]
+Description=One Convergence's NFP Proxy
+After=syslog.target network.target
+
+[Service]
+ExecStart=source /opt/stack/gbp/devstack/lib/nfp;namespace_delete /home/stack/devstack;namespace_create /home/stack/devstack 11.0.0.3
```

Comment:
Jagadish, what will be the location for namespace ? For now I just copied it the way we run it in command line but need to add the correct location.